images/all-in-one/Dockerfile

```diff
@@ -1,40 +1,54 @@
-ARG GOLANG_VERSION=1.17.8
-ARG BAZEL_VERSION=2.2.0
-ARG IBM_CLOUD_VERSION=2.5.0
+ARG GOLANG_VERSION=1.18.6
+ARG IBM_CLOUD_VERSION=2.10.0
+ARG TERRAFORM_PROVIDER_IBM_VERSION=1.39.1
+ARG TERRAFORM_PROVIDER_NULL_VERSION=3.1.0
 FROM golang:$GOLANG_VERSION AS builder
 
-RUN go get rsc.io/goversion \
+RUN go install rsc.io/goversion@v1.2.0 \
     && GO111MODULE=on
 
```

Comment:
As mentioned by golang toolchain team member [here](https://github.ibm.com/powercloud/container-dev/issues/1573#issuecomment-44085992), should we consider setting `GOPPC64=power9`?

Reply:
That won't make much difference here!

Note: I also need to remove that `&& GO111MODULE=on` which is not required.